src/Unity.Mathematics/Noise/psrdnoise2D.cs

```diff
@@ -421,16 +431,18 @@ public static float srnoise(float2 pos, float rot)
             return 11.0f * n;
         }
 
-        //
-        // 2-D non-tiling simplex noise with fixed gradients,
-        // without the analytical derivative.
-        // This function is implemented as a wrapper to "srnoise",
-        // at the math.minimal math.cost of three extra additions.
-        // Note: if this kind of noise is all you want, there are faster
-        // GLSL implementations of non-tiling simplex noise out there.
-        // This one is included mainly for completeness and compatibility
-        // with the other functions in the file.
-        //
+        /// <summary>
+        /// 2-D non-tiling simplex noise with fixed gradients,
+        /// without the analytical derivative.
+        /// This function is implemented as a wrapper to "srnoise",
+        /// at the math.minimal math.cost of three extra additions.
+        /// Note: if this kind of noise is all you want, there are faster
+        /// GLSL implementations of non-tiling simplex noise out there.
+        /// This one is included mainly for completeness and compatibility
+        /// with the other functions in the file.
+        /// </summary>
```

Comment:
Similar issue as `cellular2x2`, summary should be one liner and the rest in remarks.